### DISCUSSION TITLE
Dbt sources asset checks

### DISCUSSION THREAD

**Discussion by dveloz2** [dagster-io/dagster#24383] [Q&A] [answered]
Hi all,

I'm familiar with [dagster-dbt asset checks functionality](https://docs.dagster.io/integrations/dbt/reference#loading-dbt-tests-as-asset-checks). 

I'm wondering if there's an easy way to integrate the dbt source tests as well?
 
`dbt test --select source:my_source`

**Comment by garethbrickman** [comment, marked as answer]
We don't currently interpret tests on dbt sources as asset checks. This request is tracked by this issue: https://github.com/dagster-io/dagster/issues/18641
